components/datetime/benches/fixtures/tests/lengths_with_zones.json

```diff
@@ -1,48 +1,16 @@
 [
```

Comment:
In these JSON files:

1. I deleted any tests that were duplicates, either within the same file or in a different file
2. I added the semantic skeleton to all remaining tests
3. In a small number of cases, I moved the test to a different part of the file to make it easier to see logically what tests were being included